dev-lua/tolua++/tolua++-1.0.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="An extension of toLua, a tool to integrate C/Cpp code with Lua"
+DESCRIPTION="tolua++ is an extension of toLua, a tool to integrate C/C++ code with \
+Lua. tolua++ includes new features oriented to c++, such as class templates."
+HOMEPAGE="https://github.com/LuaDist/toluapp"
+COPYRIGHT="2009 Ariel Manzur"
+LICENSE="MIT"
+REVISION="1"
+COMMIT_SHA="b34075b76835b778bb6b2ce0aa224afd9d182887"
+SOURCE_URI="https://github.com/LuaDist/toluapp/archive/$COMMIT_SHA.zip"
+CHECKSUM_SHA256="c69a99a4b0a2bb3decab09bd8d2473911419bef601c86bb6c00ff670f88b4bb1"
+SOURCE_DIR="toluapp-$COMMIT_SHA"
+
+ARCHITECTURES="x86_64"
+
+PROVIDES="
+	tolua++ = $portVersion
+	cmd:tolua++ = $portVersion
+	lib:libtolua++ = $portVersion
+	lib:libtoluapp = $portVersion
+	"
+REQUIRES="
+	haiku
+	lua51
```

Comment:
```diff
 REQUIRES="
-	haiku
-	lua51
+	haiku$secondaryArchSuffix
+	lib:liblua$secondaryArchSuffix >= 5.1
 	"
```
Yes, `lib:liblua$secondaryArchSuffix >= 5.1` is preferred over `lua51$secondaryArchSuffix`.